test/integration/node-specific/feature_flags.test.ts

```diff
@@ -42,4 +45,35 @@ describe('Feature Flags', () => {
       });
     }
   });
+
+  describe('@@mdb.enableMongoLogger', () => {
+    let cachedEnv;
+    const loggerFeatureFlag = Symbol.for('@@mdb.enableMongoLogger');
+
+    before(() => {
+      cachedEnv = process.env;
+      process.env['MONGODB_LOG_COMMAND'] = 'emergency';
+    });
+
+    after(() => {
+      process.env = cachedEnv;
+    });
+
+    it('should instantiate a MongoLogger when set to true', () => {
+      const client = new MongoClient('mongodb://localhost:27017', { [loggerFeatureFlag]: true });
+      expect(client.mongoLogger).to.be.instanceOf(MongoLogger);
+    });
+
+    it('should not instantiate a MongoLogger when set to false', () => {
```

Comment:
I think we should make the property not exist when set to false or not set

Reply:
capturing the team convo from slack: the mongoLogger should always be present but should no op when the feature flag isn't true and logging isn't enabled